test/arr.js

```
@@ -6,33 +6,33 @@
+  )
 
-    await t.test(
-        'Test if the original output from Arr is a instance of Array',
-        () => {
-            assert.strictEqual(original instanceof Array, true)
-        }
-    )
+  await t.test(
+    'Test if the original output from Arr is a instance of Array',
+    () => {
+      assert.strictEqual(original instanceof Array, true)
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)